card-set/src/card-component.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import './Card.css';
+
+const UserCard = (props) => {
+  const [avatar, setAvatar] = React.useState('https://media.tenor.com/images/e549f9798674301c5af2c91581194091/tenor.gif')
+  const [firstName, setFirstName] = React.useState('Loading...')
+  const [lastName, setLastName] = React.useState('Loading...')
+  const [email, setEmail] = React.useState('Loading...')
+  
+  React.useEffect(() => {
```

Comment:
Completely stylistic choice here, but I tend to prefer to import my hooks from React instead of referencing it because I don't like typing `React.` over and over again. I know a lot of folks prefer to just import the default and access the properties they need off of it though, so totally your call!